config/src/main/java/org/springframework/security/config/annotation/web/builders/HttpSecurity.java

```diff
@@ -3290,20 +3290,26 @@ public HttpSecurity regexMatcher(String pattern) {
 	 */
 	public final class MvcMatchersRequestMatcherConfigurer extends RequestMatcherConfigurer {
 
+		private final List<MvcRequestMatcher> mvcMatchers;
```

Comment:
I imagine that this problem exists in other parts of Spring Security Config, like `ChannelRequestMatcherRegistry`/`MvcMatchersRequiresChannelUrl` and `StandardInterceptUrlRegistry`/`MvcMatchersAuthorizedUrl`.

If it doesn't, then I wonder if those indicate a pattern that `RequestMatcherConfigurer`/`MvcMatchersRequestMatcherConfigurer` should follow. Or if it does, let's fix those classes at the same time.